functions.php

```diff
@@ -100,6 +100,18 @@ function getUser($nick, $conn = null, $client = null) {
   return $user;
 }
 
+function deleteUser($nick, $conn = null, $client = null) {
+  if ($conn) {
+    $sql = "delete from users where login = '$nick'; ";
```

Comment:
I agree, this is not the most optimal solution. I should change this to a `sprintf` statement, but the after this: `$stmt = $conn->prepare($sql);` should take care of this issue I believe.